### ISSUE TITLE
Upgrade MongoDB Version

### ISSUE THREAD

**Issue by Rishi556** [vsc-eco/vsc-node#40]
https://github.com/vsc-eco/vsc-node/blob/eacccf9ebbb59887541af81529d12d94cb7e3126/docker-compose.yml#L31

MongoDB 4.4 is EOL(see https://endoflife.date/mongodb). Upgrading to 7.0 should be a simple change assuming no removed feature is in use.

**Comment by vaultec81** [comment]
The reason why it's at 4.4 in the first place is mongodb dropped support for Raspberry pis in latest versions. However, that can be revisited again. I'd like to have raspberry pi support but it's not the most critical thing in the world.

**Comment by vaultec81** [comment]
This might be an option https://github.com/themattman/mongodb-raspberrypi-binaries

**Comment by Rishi556** [comment]
> dropped support for Raspberry pis

Is there many running on PIs? There seem to be better options for computing at that price range these days. I think not running EOL software where possible is a higher priority.

**Comment by Geo25rey** [comment]
I agree that running an out of date version of MongoDB is not ideal. If you'd like to make a Docker image that compiles MongoDB for both arm64 v8.0 and amd64, we'd be willing to consider upgrading. However, the database we use is not a very high priority for at the moment.

**Comment by Rishi556** [comment]
Reopened as I talked with Vaultec who said this should not have been closed.
